Formula/go-jira.rb

```diff
@@ -17,6 +17,17 @@ class GoJira < Formula
   def install
     system "go", "build", "-ldflags", "-s -w", "-trimpath", "-o", bin/"jira", "cmd/jira/main.go"
     prefix.install_metafiles
+
+    # Install zsh completion
+    begin
+      output_zsh = Utils.safe_popen_read("#{bin}/jira", "--completion-script-zsh").strip
+    rescue ErrorDuringExecution => e
+      output_zsh = e.output.first.second.to_s.strip
+    end
+    # Strip out unneeded zsh module initialization and tell compinit to autoload
```

Comment:
Let's not do that: either the program provides a valid completion, and we use it. Or it should be reported to them and they fix it. But we don't want to have to mess with this.

Reply:
They should also fix the script reporting an error then.